src/test/java/plugins/AntPluginTest.java

```diff
@@ -23,15 +32,74 @@ public class AntPluginTest extends AbstractJUnitTest {
     private static final String INSTALL_VERSION_1_10 = "1.10.5";
     private static final String INSTALL_NAME_1_10 = "ant_" + INSTALL_VERSION_1_10;
 
+    private static final String NATIVE_ANT_NAME = "native_ant";
+
+    private static final String OK_PROP1 = "okPROP1=foo_bar_ok_1";
+    private static final String OK_PROP2 = "okPROP2=foo_bar_ok_2";
+    private static final String NOK_PROP1 = "nokPROP1=foo_bar_nok_1";
+    private static final String NOK_PROP2 = "nokPROP2=foo_bar_nok_2";
+    private static final String PROPERTIES = OK_PROP1+"\n"+OK_PROP2+"\n"+NOK_PROP1+"\n"+NOK_PROP2;
+    private static final String OPTS = "-showversion";
+    private static final String BUILD_FILE = "custom-build-file.xml";
+    private static final String FAKE_BUILD_FILE = "fake.xml";
+    private static final String SCRIPT_PIPELINE_ANT = "node {\n" +
+            "    withAnt(installation: '" + NATIVE_ANT_NAME + "') {\n" +
+            "        if (isUnix()) {\n" +
+            "            sh \"ant -version\"\n" +
+            "        } else {\n" +
+            "            bat \"ant -version\"\n" +
+            "        }\n" +
+            "    }\n" +
+            "}";
     FreeStyleJob job;
+    private AntBuildStep step;
+
+    public static final String REMOTE_FS = "/tmp";
+
+    @Inject private DockerContainerHolder<SshAgentContainer> docker;
+
+    private SshAgentContainer sshd;
+    private DumbSlave slave;
 
     @Before
     public void setUp() {
         job = jenkins.jobs.create(FreeStyleJob.class);
+        sshd = docker.get();
+
+    }
+
+    private SshSlaveLauncher configureDefaultSSHSlaveLauncher() {
+        return configureSSHSlaveLauncher(sshd.ipBound(22), sshd.port(22));
+    }
+
+    private SshSlaveLauncher configureSSHSlaveLauncher(String host, int port) {
```

Comment:
as this (and its caller) are probably used elsewhere - it would probably be refactored to be a helper method in the `SshAgentContainer` so it is easier to reuse across other tests.

ie - this plugins and probably the SSHPluginTest should call something like `sshd.configureSSHLauncher(agent)` at least for the host/port combination - the verification strategy and credentials may be a bit more test specific?

this way any changes to the ssh launcher can be updated in a single place and not in every test.